### PR TITLE
Use logits in CategoricalActorCriticNet

### PR DESCRIPTION
**Pull request by wassname**
A really minor change. Should we input logits instead of probs into `Categorical`? It will do less converting from logits and back, leading to increased stability during training (theoretically).

### CODE REVIEW

**Comment by ShangtongZhang** [comment]
Thanks for the contribution! We should use `logits`.
